apps/web/pages/_app.tsx

```diff
@@ -16,6 +22,15 @@ import ClientAnalyticsScript from '../src/components/ClientAnalyticsScript/Clien
 import { cookieManagerConfig } from '../src/utils/cookieManagerConfig';
 import { useSprig } from 'apps/web/src/utils/hooks/useSprig';
 
+const config = getDefaultConfig({
+  appName: 'Base.org',
+  projectId:
+    process.env.NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID ?? '99a7649b4421d35d175fbca94c1bd880',
```

Comment:
do we need the hardcoded id here?

Reply:
Build kite seems to break without it even though we have the ENV var set